test/utils/transformAwareTranslatorTests.ts

```diff
@@ -7,73 +7,33 @@ import * as Plottable from "../../src";
 import * as TestMethods from "../testMethods";
 
 describe("Translator", () => {
-    it("getTranslator() creates only one Translator per html root", () => {
-        const div = TestMethods.generateDiv();
-        const component = new Plottable.Component();
-        sinon.stub(component, "rootElement").returns(div);
-
-        const t1 = Plottable.Utils.getTranslator(component);
-        assert.isNotNull(t1, "created a new Translator on a <svg>");
-        const t2 = Plottable.Utils.getTranslator(component);
-        assert.strictEqual(t1, t2, "returned the existing Translator if called again with same <svg>");
-
-        div.remove();
-      });
-
-    it("converts points to html-space correctly", () => {
-        const div = TestMethods.generateDiv();
```

Comment:
No longer relevant; we don't have notions of "html-space" vs "svg-space" (it's just "Component space")